JSDemos/Demos/TabPanel/Overview/React/styles.css

```diff
@@ -1,40 +1,147 @@
-.tabpanel-item {
-  height: 200px;
-  user-select: none;
-  padding-left: 25px;
-  padding-top: 55px;
+body {
+  margin: 0;
+  padding: 0;
 }
 
-.mobile .tabpanel-item {
-  padding-top: 10px;
+#tabpanel-demo {
+  display: flex;
+  min-height: 450px;
+  background-color: rgba(191, 191, 191, 0.15);
 }
 
-.tabpanel-item > div {
-  float: left;
-  padding: 0 85px 10px 10px;
+.widget-container {
+  display: inline;
+  flex-grow: 1;
+  min-width: 648px;
+  padding: 16px 32px;
 }
 
-.tabpanel-item p {
-  font-size: 16px;
-  margin: 0;
+.dx-tabpanel {
+  border-radius: 8px;
```

Comment:
I think adding additional customizations to the component should be careful, because there will be problems when switching themes and sizes.